src/Http/TracingHttpClient.php

```diff
@@ -119,6 +119,7 @@ public function request(string $method, string $url, array $options = []): Respo
 
                     $span->setAttribute(TraceAttributes::HTTP_STATUS_CODE, $info['http_code']);
                     $span->setAttribute(TraceAttributes::HTTP_URL, $info['url']);
+                    $span->addEvent('HTTP headers received');
```

Comment:
Do we have the curl level timestamp for this information, eg. https://www.php.net/manual/fr/function.curl-getinfo.php